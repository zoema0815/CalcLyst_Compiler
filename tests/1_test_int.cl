int main() {
    int a;
    int b;
    a = 40;
    b = 20;
    printInt(a);
    printInt(b);
    /*Binary Operation*/
    printInt(a + b);
    printInt(a - b);
    printInt(a * b);
    printInt(a / b);
    return 0;
}