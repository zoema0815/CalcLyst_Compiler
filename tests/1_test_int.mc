int main() {
    int a;
    a = 4096;
    printInt(a);
    return 0;
}