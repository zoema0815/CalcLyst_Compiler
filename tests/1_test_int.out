40
20
60
20
800
2
